src/lib/Components/Consignments/Components/DoneButton.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react"
+
+import BottomAlignedButton from "./BottomAlignedButton"
+
+export interface DoneButtonProps {
+  onPress: () => void
+  children?: any[]
+}
+
+const render = (props: DoneButtonProps) => {
+  const doneButtonStyles = {
+    backgroundColor: "black",
+    marginBottom: 0,
+    paddingTop: 18,
+    height: 56,
+  }
```

Comment:
These should be consistent - so either all of them gets the change, or they all stay the same 👍 

I wonder if that difference is the spell checker bar?